sources/airtable/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Airtable
+description: dlt source for Airtable.com API
+keywords: [airtable api, airtable source, airtable, base]
+---
+
+
+# Airtable
+
+[Airtable](https://airtable.com/) is a spreadsheet-database hybrid, with the features of a database but applied to a spreadsheet. It is also marketed as a low‒code platform to build next‒gen apps. Spreadsheets are called *airtables* and are grouped into *bases*. 
+Records can link to each other.
+
+This [dlt source](https://dlthub.com/docs/general-usage/source) creates a [dlt resource](https://dlthub.com/docs/general-usage/resource) for every airtable and loads it into the destination. 
+
+
+## Supported methods to load airtables
+
+1. identify the *base* you want to load from. The ID starts with "app". See [how to find airtable IDs](https://support.airtable.com/docs/finding-airtable-ids)
+2. identify the *airtables* you want to load. You can identify in three ways:
+
+| filter             | Description                                              |
+| ---------------- | -----------------------------------------------------------|
+| table_names  | retrieves *airtables* from a given *base* for a list of user-defined mutable names of tables |
+| table_ids    | retrieves *airtables* from a given *base* for a list of immutable IDs defined by airtable.com at the creation time of the *airtable*. IDs start with "tbl". See [how to find airtable IDs](https://support.airtable.com/docs/finding-airtable-ids) |
+| empty filter | retrieves all *airtables* from a given *base* |
+
+
+## Supported write dispositions
+This connector supports the write disposition `replace`, i.e. it does a [full load](https://dlthub.com/docs/general-usage/full-loading) on every invocation.
+We do not support `append`, i.e. [incremental loading](https://dlthub.com/docs/general-usage/incremental-loading) because a *base* can contain only [up to 50k records in the most expensive plan](https://support.airtable.com/docs/airtable-plans).
+
+If resource consumption for data loading becomes a concern in practice [request](https://github.com/dlt-hub/verified-sources/issues/new/choose) the `append` loading method.
```

Comment:
I'd rather mention that you can change the write disposition after the resource is created ie. to append or merge + incremental load and here it is how: https://dlthub.com/docs/general-usage/resource#adjust-schema